<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispositivosDialog</class>
 <widget class="QDialog" name="DispositivosDialog">
  <property name="windowTitle">
   <string>Dispositivos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutInputs">
     <item>
      <widget class="QComboBox" name="comboCliente"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMarca"/>
     </item>
    <item>
      <widget class="QLineEdit" name="lineEditModelo"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIMEI"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSerie"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColor"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAccesorios"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAgregar">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableDispositivos">
     <property name="columnCount">
      <number>7</number>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Cliente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modelo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IMEI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NÂº Serie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Accesorios</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
